showcase/app/templates/components/form/super-select.hbs

```diff
@@ -0,0 +1,1020 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+{{page-title "Form::SuperSelect Components"}}
+
+<BasicDropdownWormhole />
+
+<Shw::Text::H1>Form::SuperSelect</Shw::Text::H1>
+
+<section data-test-percy>
+  <Shw::Text::H2>Form::SuperSelect::Single::Base</Shw::Text::H2>
+
+  <Shw::Text::H3>Interaction status</Shw::Text::H3>
+
+  <Shw::Grid @columns={{3}} as |SG|>
+    <SG.Item @label="Default">
+      <Hds::Form::SuperSelect::Single::Base
+        @onChange={{this.noop}}
+        @options={{@model.OPTIONS}}
+        @ariaLabel="Label"
+        as |option|
+      >
+        {{option}}
+      </Hds::Form::SuperSelect::Single::Base>
+    </SG.Item>
+    <SG.Item @label="With placeholder">
+      <Hds::Form::SuperSelect::Single::Base
+        @onChange={{this.noop}}
+        @options={{@model.OPTIONS}}
+        @placeholder="Placeholder text"
+        @ariaLabel="Label"
+        as |option|
+      >
+        {{option}}
+      </Hds::Form::SuperSelect::Single::Base>
+    </SG.Item>
+    <SG.Item @label="Selected">
+      <Hds::Form::SuperSelect::Single::Base
+        @onChange={{fn (mut @model.SELECTED_OPTION)}}
+        @options={{@model.OPTIONS}}
+        @selected={{@model.SELECTED_OPTION}}
+        @ariaLabel="Label"
+        as |option|
+      >
+        {{option}}
+      </Hds::Form::SuperSelect::Single::Base>
+    </SG.Item>
+  </Shw::Grid>
+
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>States</Shw::Text::H3>
+
+  <Shw::Grid @columns={{3}} as |SG|>
+    {{#let (array "base" "invalid" "disabled") as |variants|}}
+      {{#each @model.STATES as |state|}}
+        <SG.Item>
+          <Shw::Flex @direction="column" as |SF|>
+            {{#each variants as |variant|}}
+              {{! template-lint-disable simple-unless }}
+              {{#unless (and (eq variant "disabled") (eq state "focus"))}}
+                <SF.Item @label="{{capitalize variant}} / {{capitalize state}}">
+                  <Hds::Form::SuperSelect::Single::Base
+                    @onChange={{fn (mut @model.SELECTED_OPTION)}}
+                    @options={{@model.OPTIONS}}
+                    @selected={{@model.SELECTED_OPTION}}
+                    @disabled={{if (eq variant "disabled") true}}
+                    @isInvalid={{if (eq variant "invalid") true}}
+                    @ariaLabel="Label"
+                    mock-state-value={{state}}
+                    as |option|
+                  >
+                    {{option}}
+                  </Hds::Form::SuperSelect::Single::Base>
+                </SF.Item>
+              {{/unless}}
+            {{/each}}
+          </Shw::Flex>
+        </SG.Item>
+      {{/each}}
+    {{/let}}
+  </Shw::Grid>
+
+  <Shw::Divider @level="2" />
+
+  <Shw::Text::H3>List</Shw::Text::H3>
```

Comment:
When I checked this morning this looked like this:
<img width="1073" alt="screenshot_3761" src="https://github.com/hashicorp/design-system/assets/686239/a8efe706-1ee3-4279-ac83-fed60a360365">
Now it seems the "disabled" state has disappeared
<img width="1067" alt="screenshot_3767" src="https://github.com/hashicorp/design-system/assets/686239/3f878476-d8c8-437a-bab4-31ba2ead8cfa">

Is it intended? For me it was useful to see also the disable visual state:

Reply:
Introducing the `disabled` option created the `[Object]` issue flagged by you above. I reduced the `disabled` examples to only cases where the options are objects (not strings) in 4969b1e. We can revisit this and or introduce more examples.

Reply:
Added a commit, based on your work, showcasing main elements and subcomponents, including the disabled state.